src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.scss

```diff
@@ -3,3 +3,12 @@
   height: 310px;
   padding: 40px;
 }
+
+.MigrationPoliciesEmptyState {
+  &__button {
+    .pf-v5-c-button.pf-m-primary {
+      color: var(--pf-v5-c-button--disabled--Color);
+      background-color: var(--pf-v5-c-button--disabled--BackgroundColor);
+    }
+  }
+}
```

Comment:
Not needed, we can get this from PF

Reply:
We need it. We need to keep the button clickable while having the style of not clickable one. Because of the design. To achieve this, we need to style the button. Otherwise it won't be clickable and no any popover could be rendered. Same for hovering on the button, unfortunately.

Reply:
This can be achieved by using isAriaDisabled prop as I suggested in the code for MigrationPoliciesCreateButton.tsx

Reply:
Will try it out, thanks a lot, Aviv! :)